### ISSUE TITLE
require('http') is typed incorrectly

### ISSUE THREAD

**Issue by rileylark** [facebook/flow#473]
Given

```
/* @flow */

var http = require('http');

var requestListener: any = {};

http.createServer(requestListener);
```

I'd expect flow to either think `http` has type `any` or else know about the methods it exposes, but flow reports an error:

```
flowTest.js:7:1,34: call of method createServer
Could not resolve name
```

I could be misunderstanding something. In the meantime my only workaround is to manually mark http as type `any`.

**Comment by samwgoldman** [comment]
The fix for this issue has already landed in 7254f8b49655540c973e8ff11fc9effad3c2e3e8, but that hasn't yet been released.

**Comment by rileylark** [comment]
Awesome, thanks!
